src/utils/crypto_utils/tests/CMakeLists.txt

```diff
@@ -10,10 +10,16 @@ disablertti ()
 set (sources main.cpp crypto_utils_ut.cpp)
 
 find_package (Catch2 REQUIRED)
+find_package (OpenSSL REQUIRED)
 
 add_executable (${PROJECT_NAME} ${sources})
 
-target_link_libraries (${PROJECT_NAME} PRIVATE aduc::crypto_utils aduc::string_utils Catch2::Catch2)
+message(OPENSSL_CRYPTO_LIBRARY="${OPENSSL_CRYPTO_LIBRARY}")
```

Comment:
Need to remove. Not needed and already stated by crypto_utils.

Reply:
will send a separate PR for these.